lib/taurus/core/taurusmanager.py

```diff
@@ -108,27 +112,59 @@ def cleanUp(self):
 
         self._thread_pool.join()
         self._thread_pool = None
+        self._sthread_pool.join()
+        self._sthread_pool = None
 
         self._state = ManagerState.CLEANED
 
     def addJob(self, job, callback=None, *args, **kw):
-        """Add a new job (callable) to the queue. The new job will be processed
-        by a separate thread
+        """ Deprecated. Wrapper of enqueueJob. See enqueueJob documentation.
+        """
+        self.deprecated(dep='addJob', alt='enqueueJob', rel='4.3.2')
+        self.enqueueJob(job, callback, args, kw)
 
+    def enqueueJob(self, job, callback=None, jobargs=None, jobkwargs=None,
+                   serialization_mode=None):
+        """ Enqueue a job (callable) to the queue. The new job will be
+        processed by a separate thread
         :param job: (callable) a callable object
         :param callback: (callable) called after the job has been processed
-        :param args: (list) list of arguments passed to the job
-        :param kw: (dict) keyword arguments passed to the job
+        :param jobargs: (list) list of arguments passed to the job
+        :param jobkwargs: (dict) keyword arguments passed to the job
+        :param serialization_mode:
+        (taurus.core.taurusbasetypes.TaurusSerializationMode) serialization
+        mode
         """
-        if self._serialization_mode == TaurusSerializationMode.Concurrent:
+        if jobargs is None:
```

Comment:
Why not putting `jobargs=()` in the method signature? It would be clearer and shorter